microsoft/botframework-solutions
# [Enterprise Bot Generator][Typescript] Create a Dialog Generator

## Description
- Add the dialog generator.

## Related Issue
Fix #658 

## Testing Steps

1. Go to `AI\templates\Enterprise-Template\src\typescript\generator-botbuilder-enterprise` and run `npm link` to symlink the package folder
2. Open a terminal wherever you want to create a dialog
3. Run `yo botbuilder-enterprise:dialog`
4. Follow the generator's prompts
5. Check that the dialog was created successfully

## Checklist
~- [ ] I have commented my code, particularly in hard-to-understand areas~
~- [ ] I have added or updated the appropriate unit tests~
~- [ ] I have tested any new/updated dialogs using Speech in the emulator to ensure the [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) property is set to enable a high quality speech-first experience and the appropriate [InputHints](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) are set correctly.~
~- [ ] I have updated related documentation~

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
~- [ ] A duplicate issue is filed to track future  work.~

If you have updated responses or `.lu` files:
~- [ ] All languages have been updated~
~- [ ] You have tested deployment with your new models~
